bench/README.md

```diff
@@ -0,0 +1,29 @@
+# Ecto Benchmarks
+
+Ecto has a benchmark suite to track performance of sensitive operations. Benchmarks
+are run using the [Benchee](https://github.com/PragTob/benchee) library and
+need `Postgres` and `MySQL` up and running.
+
+To run the benchmarks tests just type in the console:
+
+```
+# POSIX-compatible shells
+$ BENCHMARKS_OUTPUT_PATH=bench/results MIX_ENV=bench mix run bench/bench_helper.exs
+```
+
+```
+# other shells
+$ env BENCHMARKS_OUTPUT_PATH=bench/results MIX_ENV=bench mix run bench/bench_helper.exs
+```
+
+Benchmarks are inside the benchmarks/ directory and are divided into two
```

Comment:
Minor nitpicks: 

1. I wouldn't use backticks for Potgres and MySQL:

```
and need PostgreSQL and MySQL up and running.
```

Also, I would use "PostgreSQL" instead of "Postgres

2. I would use backticks for the dir:

```
Benchmarks are inside the `benchmarks/` directory
```

3. And maybe use bullet points for the list of categories?

```
* `micro benchmarks`: Operations that don't actually interface with the database,
but might need it up and running to start the Ecto agents and processes.
* `macro benchmarks`: Operations that are actually run in the database. This are
more likely to integration tests.
```

Reply:
Thanks for your considerations! :heart: